flintrock/ec2.py

```diff
@@ -506,10 +507,11 @@ def launch(
                 v=vpc_id))
 
     try:
-        security_groups = get_or_create_ec2_security_groups(
+        flintrock_security_groups = get_or_create_ec2_security_groups(
             cluster_name=cluster_name,
             vpc_id=vpc_id,
             region=region)
+        security_group_ids = list(security_groups) + [sg.id for sg in flintrock_security_groups]
```

Comment:
Surprisingly, this still works if you provide a name (as opposed to an ID) for the additional security groups.